scripts/roboclawtest.py

```diff
@@ -9,17 +9,23 @@
 import sys
 from os import path
 # need to add the roboclaw.py file in the path
-sys.path.append(path.join(path.expanduser('~'), 'osr_ws/src/osr-rover-code/ROS/osr_control/osr_control'))
+sys.path.append(path.join(path.expanduser('~'), 'osr2_ws/src/osr-rover-code/ROS/osr_control/osr_control'))
```

Comment:
why'd you change this? The instructions all have `osr_ws`, https://github.com/nasa-jpl/osr-rover-code/blob/foxy-devel/setup/rpi.md   ?

Reply:
```suggestion
sys.path.append(path.join(path.expanduser('~'), 'osr_ws/src/osr-rover-code/ROS/osr_control/osr_control'))
```